### ISSUE TITLE
Path modification bash shell script

### ISSUE THREAD

**Issue by rickstaa** [alebcay/awesome-shell#361]
Not sure if I'm in the right place but I made a small bash alias wrapper script which I thought might be useful to other people. It can be used to easily list, remove, move and add paths to the PATH and PYTHONPATH environmental variables. Feel free to add it to the list if you consider it useful [https://github.com/rickstaa/.pathmod](https://github.com/rickstaa/.pathmod). I am planning to add support for other shells in the future. Contributions are always welcome.

**Comment by MichaelHabib** [comment]
Hi,
Is this something like Direnv ?

**Comment by rickstaa** [comment]
Hey @MichaelHabib. Thanks for the tip I did not know about Direnv. It has a lot of useful functions! I looked at the documentation, and regarding PATH modification tools, my script has some additional options (remove, move and check PATH and PYTHONPATH) compared to Direnv. It further misses the expand path option. Hope you find my scrip useful. If you have suggestions or experience problems with my script let me know.
